src/engine/Graphics2d.js

```diff
@@ -691,18 +691,40 @@ class Graphics2d extends React.Component {
     this.m_sliceRatio = this.props.sliderValue;
     this.m_mode2d = this.props.mode2d;
 
+    const isLandscape = window.matchMedia('(orientation: landscape)').matches;
+
     const wrapperStyles = {
-      display: 'flex',
-      justifyContent: 'flex-start',
-      width: `calc(100% - 110px)`,
-      height: `calc(100% - 110px)`,
-      margin: '100px 125px',
+      width: '100%',
+      height: '100%',
+      justifyContent: 'center',
+      overflow: 'scroll',
+      padding: '0',
     };
+
     const canvasStyles = {
-      maxWidth: '100%',
+      display: 'block',
+      width: '150vw',
+      height: '100%',
+      margin: '0',
+      padding: '0',
       border: '2px solid #dc5e47',
       borderRadius: '10px',
     };
+
+    if (window.innerWidth < 768) {
```

Comment:
why we cannot add these styles using CSS only?

Reply:
Great suggestion!Will do it